Plugins/Public/playercntl_plugin/Main.cpp

```diff
@@ -1566,6 +1566,30 @@ bool ExecuteCommandString_Callback(CCmds* cmds, const wstring &wscCmd)
 		Rename::ReloadLockedShips();
 		return true;
 	}
+	else if (IS_CMD("sethp"))
+	{
+		returncode = SKIPPLUGINS_NOFUNCTIONCALL;
+		MiscCmds::AdminCmd_SetHP(cmds, cmds->ArgUInt(1), cmds->ArgStr(2));
```

Comment:
ArgCharname/ArgStrToEnd?

Reply:
Player names can contain no spaces so ArgStrToEnd doesn't seem to add anything.
ArgCharname returns a weirdly formatted string "id 5" that I'd need to deconstruct to get anything useful out of. It honestly didn't seem useful.